RecipesManagerWebClient.Web/Pages/Recipes/Recipe.cshtml

```diff
@@ -0,0 +1,227 @@
+                }
+            </div>
+        } 
+        else{
+            <div class="dish-picture">
+                <img src="/assets/no-image.svg" class="dish-image" alt="No Image" />
+                <div class="calories-time-details">
+                    <div class="calories">
+                        <img b-c3mbu1qqil src="~/assets/calories.svg">
+                        <div class="calories-number">@Model.Recipe.Calories calories</div>
+                    </div>
+                    <div class="time">
+                        <img b-c3mbu1qqil src="~/assets/stopwatch.svg" />
+                        <div class="dish-time">@Model.Recipe.MinutesToCook min</div>
+                    </div>
+                    @if (Model.Recipe.IsSaved)
+                    {
+                        <div class="dish-saved"><img b-c3mbu1qqil src="~/assets/saved.svg" /></div>
+                    }
+                    else
+                    {
+                        <div class="dish-saved"><img b-c3mbu1qqil src="~/assets/unsaved.svg" /></div>
+                    }
+                </div>
+            </div>
+        }
+    </div>
+    <div class="dish-details">
+        <div class="author">
+            <div class="profile">
+                <div class="photo">
+                    <img src="/assets/avatar.svg" class="avatar" alt="profile_photo" />
+                </div>
+                <div class="author-description">
+                    <p class="name">@Model.Recipe.CreatedBy.Name</p>
+                    <p class="followers">13 subscribers</p>
+                </div>
+            </div>
+            <div class="follow-btn">
+                <button class="create">Follow</button>
+            </div>
+        </div>
+        <div class="dish-description">
+            <p class="dish-title">@Model.Recipe.Name</p>
+            <p class="dish-category">@Model.Recipe.Categories.FirstOrDefault()?.Name</p>
+            
+        </div>
+           
+    </div>
+
+    <div class="btn">
+        <div class="ingredients-overview">
+            <div class="title desktop-ingr-btn">Ingredients</div>
+            <div class="ingredients-number">@Model.Recipe.IngredientsText.Split('.').Length Items</div>
+
+            <button class="title switch-buttons" id="btn1">Ingredients</button>
+        </div>
+        <div class="procedure-overview">
+            <div class="title desktop-proc-btn">Procedure</div>
+            <button class="title switch-buttons" id="btn2">Procedure</button>
+        </div>
+    </div>
+
+    <div class="ingredients-procedure">
+        <div id="mobile-ingredients" class="ingredients-list">
+            <div class="list">
+                <!--foreach (var ingredient in Model.Recipe.Ingredients){
+                    <div>ingredient.Name</div>
+                    <div>ingredient.Quantity ingredient.Unit</div>
+                } -->
+                @if (!string.IsNullOrEmpty(Model.Recipe.IngredientsText))
```

Comment:
If ingredients array is not empty show ingredients from array otherwise display plain text, dont split by dot